tests/CookingConvertor/Cooking.java

```diff
@@ -0,0 +1,156 @@
+                case Units.TBSP:
+                    return TBSP;
+                default:
+                return 0;
+            }
+        }
+
+        public static float getTSP() {
+            return TSP;
+        }
+
+        public static float getML() {
+            return ML;
+        }
+
+        public static float getOZ() {
+            return OZ;
+        }
+
+        public static float getPOUND() {
+            return POUND;
+        }
+
+        public static float getGALLON() {
+            return GALLON;
+        }
+
+        public static float getQUART() {
+            return QUART;
+        }
+
+        public static float getPINT() {
+            return PINT;
+        }
+
+        public static float getCUP() {
+            return CUP;
+        }
+
+        public static float getFlOunce() {
+            return FL_OUNCE;
+        }
+
+        public static float getTBSP() {
+            return TBSP;
+        }
+
+        public static float switching (CookingMeasurement originalMeasurement,String unit) {
+            float newAmount = 0;
+            switch (unit.toUpperCase()) {
+                case "TSP":
+                    newAmount = (value(originalMeasurement.getUnit()) / getTSP()) * originalMeasurement.getAmount();
+                    break;
+                case "OZ":
+                    newAmount = (value(originalMeasurement.getUnit()) / getOZ()) * originalMeasurement.getAmount();
+                    break;
+                case "POUND":
+                    newAmount = (value(originalMeasurement.getUnit()) / getPOUND()) * originalMeasurement.getAmount();
+                    break;
+                case "QUART":
+                    newAmount = (value(originalMeasurement.getUnit()) / getQUART()) * originalMeasurement.getAmount();
+                    break;
+                case "GALLON":
+                    newAmount = (value(originalMeasurement.getUnit()) / getGALLON()) * originalMeasurement.getAmount();
+                    break;
+                case "PINT":
+                    newAmount = (value(originalMeasurement.getUnit()) / getPINT()) * originalMeasurement.getAmount();
+                    break;
+                case "CUP":
+                    newAmount = (value(originalMeasurement.getUnit()) / getCUP()) * originalMeasurement.getAmount();
+                    break;
+                case "FL_OUNCE":
+                    newAmount = (value(originalMeasurement.getUnit()) / getFlOunce()) * originalMeasurement.getAmount();
+                    break;
+                case "TBSP":
+                    newAmount = (value(originalMeasurement.getUnit()) / getTBSP()) * originalMeasurement.getAmount();
+                    break;
+                case "ML":
+                    newAmount = (value(originalMeasurement.getUnit()) / getML()) * originalMeasurement.getAmount();
+                    break;
+                default:
+                    System.out.println("Enter again");
+            }
```

Comment:
Redundant.
Use a case to just get the right amount, move the switch statement to Amounts class, leave only the formula here.